src/main/io/ledstrip.c

```diff
@@ -444,14 +444,18 @@ static void applyLedFixedLayers()
     for (int ledIndex = 0; ledIndex < ledCounts.count; ledIndex++) {
         const ledConfig_t *ledConfig = &ledStripConfig()->ledConfigs[ledIndex];
         hsvColor_t color = *getSC(LED_SCOLOR_BACKGROUND);
+	hsvColor_t color2 = *getSC(LED_SCOLOR_BACKGROUND); //next color above the one selected, or color 0 if your are at the maximum
+	hsvColor_t color0 = *getSC(LED_SCOLOR_BACKGROUND); //Previous color to the one selected, modulo color count
```

Comment:
What is the meaning of `color`, `color0`, `color2` in this context? I'd use names that indicate what these variables are used for.